ts/store/actions/content.ts

```diff
@@ -44,7 +45,14 @@ export const loadContextualHelpData = createAsyncAction(
   "LOAD_CONTEXTUAL_HELP_TEXT_DATA_FAILURE"
 )<void, ContextualHelp, Error>();
 
+export const loadIdps = createAsyncAction(
+  "LOAD_IDPS_REQUEST",
+  "LOAD_IDPS_SUCCESS",
+  "LOAD_IDPS_FAILURE"
+)<void, SpidIdps, Error>();
+
```

Comment:
With @Undermaken we decided to keep a local fallback when no remote idps data is available to avoid blocking the user's login flow, I guess that showing an error could create confusion what do you think?

Reply:
Added in 61a303a